Library/Formula/fbthrift-compiler.rb

```diff
@@ -0,0 +1,58 @@
+class FbthriftCompiler < Formula
+  homepage "https://github.com/facebook/fbthrift"
+  url "https://github.com/facebook/fbthrift/archive/v0.28.0.tar.gz"
+  version "28.0"
+  sha1 "7fed41f6d7c072b41a550fadf36ba95bdd6d9813"
+
+  head "https://github.com/facebook/fbthrift.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  depends_on "glog"
+  depends_on "boost"
+  depends_on "folly"
+
+  patch :p1, :DATA
```

Comment:
Please add a comment explaining why the patch is necessary. Did you report the issue upstream?